### DISCUSSION TITLE
How to record Log.

### DISCUSSION THREAD

**Discussion by ChenKitDon** [esnet/iperf#1566] [Q&A]
I use the iper.exe tool, but after running for a while, it will crash by itself. How to record the LOG of this problem, and I want to analyze where the LOG problem is.
System: win11

**Comment by davidBar-On** [comment]
Usually, logs are not sufficient to analyze crashes, and using tools like gdb is needed.

In any case, what is the iperf version?  Can you run `iperf.exe -v` and share the output?

**Comment by ChenKitDon** [comment]
The version is 2.0.5,Can you provide the gdb tool?

**Comment by swlars** [comment]
GDB can be found here: https://www.sourceware.org/gdb/

I'm going to move this to a discussion, since Windows isn't officially supported.

**Comment by ChenKitDon** [comment]
Thank you for providing, can you provide the SOP of GDB, let me refer to how to use and analyze the problem?

**Comment by davidBar-On** [comment]
> The version is 2.0.5

You are using iperf2 but this issues/discussions are for iperf3.  Use https://sourceforge.net/projects/iperf2/ to open iperf2 issues/discussions.

In any case, you are using a **very old** version of iperf2 that may not be compatible with win11.  I suggest first to try downloading a newer version (probably 2.1.9) of iperf2 from https://sourceforge.net/projects/iperf2/files/.  Note that both client and server should be upgraded.

Alternatively, if you planned using iperf3, a Windows version can be downloaded from https://files.budman.pw/ (not an official site).  Also, iperf2 and iperf3 are not compatible, so both server and client should be either iperf2 or iperf3.
